aspnetcore/release-notes/aspnetcore-3.0.md

```diff
@@ -0,0 +1,180 @@
+---
+title: What's new in ASP.NET Core 3.0
+author: rick-anderson
+description: Learn about the new features in ASP.NET Core 3.0.
+ms.author: riande
+ms.custom: mvc
+ms.date: 12/18/2018
+uid: aspnetcore-3.0
+---
+https://docs.microsoft.com/en-us/windows-server/security/kerberos/kerberos-authentication-overview
```

Comment:
@Rick-Anderson Is this link supposed to go somewhere in the doc?

Reply:
No, it's a reminder at the end I have a bunch of links that need  a global removal of `/https://docs.microsoft.com/en-us`